src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java

```diff
@@ -152,6 +152,27 @@ private void updateIfNeeded() {
             }
         }
 
+        // Enforce TLS.
+        Element gui = (Element) mConfig.getDocumentElement()
+                .getElementsByTagName("gui").item(0);
+        boolean tls = Boolean.parseBoolean(gui.getAttribute("tls"));
+        if (!tls) {
+            Log.i(TAG, "Enforce TLS");
+            gui.setAttribute("tls", Boolean.toString(true));
+            changed = true;
+        }
+
+        // Update deprecated 8080 port to 8384
```

Comment:
I'd say this doesn't really matter. If the port is changed for existing installs, it should be done by syncthing. And if syncthing doesn't do it, we don't need that either.